app/views/home/home.js

```diff
@@ -19,11 +19,19 @@ const openWeatherData = require('../../fixtures/openweathermap.json');
 const weatherbitData = require('../../fixtures/weatherbit.json');
 const wundergroundData = require('../../fixtures/wunderground.json');
 
-const lambda = new AWS.Lambda({
+const awsConfig = {
   region:'us-west-2',
   accessKeyId: process.env.PREACT_APP_AWS_ACCESS_KEY_ID, 
-  secretAccessKey: process.env.PREACT_APP_AWS_SECRET_ACCESS_KEY
-});
+  secretAccessKey: process.env.PREACT_APP_AWS_SECRET_ACCESS_KEY  
+}
+
+const s3Params = {
+  Bucket: 'wx-aggregator',
+  Key: 'forecastResults.json',
+}
+
+const lambda = new AWS.Lambda(awsConfig);
```

Comment:
Does this `accessKeyId` and `secretAccessKey` with the `PREACT_APP` keys need to be used to initialize an new `AWS.Lambda`? If not, just directly invoke AWS Lambda or S3 directly with just `us-west-2`